src/entities/course/index.ts

```diff
@@ -0,0 +1,2 @@
+export type { Course, CourseCardProps, CourseStatus } from './types';
```

Comment:
`CourseCardProps` isn't used outside the entity and shouldn't be exported

```suggestion
export type { Course, CourseStatus } from './types';
```

Reply:
Done